mypy/build.py

```diff
@@ -2398,12 +2398,38 @@ def skipping_ancestor(manager: BuildManager, id: str, path: str, ancestor_for: '
                           severity='note', only_once=True)
 
 
+def log_configuration(manager: BuildManager) -> None:
+    """Output useful configuration information to LOG and TRACE"""
+
+    manager.log()
+    manager.log("Mypy version %s" % __version__)
+    manager.log("Python interpreter located at: %s" % manager.options.python_executable)
```

Comment:
That's the *configured* interpreter. It may or may not match `sys.executable`. Maybe print both?